UI/ImDebugger/ImStructViewer.cpp

```diff
@@ -196,10 +223,11 @@ static constexpr int COLUMN_CONTENT = 2;
 static constexpr int MAX_POINTER_ELEMENTS = 0x100000;
 static constexpr u32 INDEXED_MEMBERS_CHUNK_SIZE = 0x1000;
 
-void ImStructViewer::Draw(MIPSDebugInterface* mipsDebug, bool* open) {
+void ImStructViewer::Draw(ImConfig& cfg, ImControl& control, MIPSDebugInterface* mipsDebug) {
```

Comment:
Just noting that we use the convention that & and * go next to the variable name, not the type.

Reply:
Sure, I can fix it in the whole file and GhidraClient in this PR. I used this style because https://www.ppsspp.org/docs/development/getting-started/ links to the "Google C++ Style Guide" which places them next to the type. Maybe it would be good to list there most important differences in PPSSPP code style?

Reply:
Thanks!

Yeah, need to clarify the style indeed.